e2e/test-page.html

```diff
@@ -10,7 +10,7 @@
       width: 100px;
       height: 100px;
       background: gray;
-      box-shadow: 0 0 5px 2px black;
+      box-shadow: 0 0 5px 2px black, 0 0 5px 2px red inset;
```

Comment:
🤔 one of this shadows is 'inner', the other one is 'normal'. If you look at `e2e/expected-layers.asketch.json` it looks like both are translated to `innerShadow` in Sketch. Is that correct behaviour?

Reply:
sorry, There was a problem to check `boxShadow` is `inset`.
I fixed it!